frontend/packages/client/src/components/ProposalCreate/StepOne/index.js

```diff
@@ -127,38 +136,195 @@ const StepOne = ({
+              className={`border-light rounded-sm is-flex is-align-items-center m-0 p-0 mb-4 cursor-pointer ${
+                voteType === 'single-choice' ? 'border-grey' : 'border-light'
+              }`}
+              onClick={() => handleVoteType('single-choice')}
+            >
+              <div className="p-4">
+                <div className="is-flex is-align-items-center mr-2">
+                  <div className="is-flex is-align-items-center mr-2">
+                    <label className="radio is-flex">
+                      <input
+                        {...register('voteType')}
+                        type="radio"
+                        value="single-choice"
+                        className="green-radio"
+                      />
+                      <span />
+                    </label>
+                  </div>
+                </div>
+              </div>
+              <div className="py-5 pr-5">
+                <h5 className="title is-6 mb-2">Single Choice Voting</h5>
+                <p>
+                  Voters can only vote on one option and all options are
+                  customized by proposal creator.
+                </p>
+              </div>
+              <div className="has-background-light-grey p-4 rounded-sm-br rounded-sm-tr is-flex is-flex-direction-column is-align-self-stretch is-justify-content-center ">
+                <div
+                  className="is-flex is-align-items-center mb-1"
+                  style={{ whiteSpace: 'nowrap' }}
+                >
+                  <div
+                    className="rounded-full has-background-grey has-text-white mr-2 is-flex is-align-items-center is-justify-content-center"
+                    style={{ width: 12, height: 12 }}
+                  ></div>
+                  <span className="smaller-text">Option A</span>
+                </div>
+                <div
+                  className="is-flex is-align-items-center mb-1"
+                  style={{ whiteSpace: 'nowrap' }}
+                >
+                  <div
+                    className="rounded-full has-background-grey has-text-white mr-2 is-flex is-align-items-center is-justify-content-center"
+                    style={{ width: 12, height: 12 }}
+                  >
+                    <span style={{ fontSize: 8 }}>&#x2713;</span>
+                  </div>
+                  <span className="smaller-text">Option B</span>
+                </div>
+                <div
+                  className="is-flex is-align-items-center"
+                  style={{ whiteSpace: 'nowrap' }}
+                >
+                  <div
+                    className="rounded-full has-background-grey has-text-white mr-2 is-flex is-align-items-center is-justify-content-center"
+                    style={{ width: 12, height: 12 }}
+                  ></div>
+                  <span className="smaller-text">Option C</span>
+                </div>
+              </div>
+            </div>
```

Comment:
could you make this a static component so we can reduce  code here soma as for the other option